### PR TITLE
Fix Minor UI Issues

### PR DESCRIPTION
**Pull request by chloeelim**
1. Fixes the alignment of the placeholder box ("No Iterations Found") when viewed on wider screens.
<table>
<tr>
<td><strong>before</strong></td>
<td><strong>after</strong></td>
</tr>

<tr>
<td>Screenshot 2022-10-12 at 1 18 37 PM</td>
<td>Screenshot 2022-10-12 at 1 19 07 PM</td>
</tr>
</table>

2. Adds back the accidentally deleted 'Iterations' header in the `CommissionDetailsPane`
<table>
<tr>
<td><strong>before</strong></td>
<td><strong>after</strong></td>
</tr>

<tr>
<td>Screenshot 2022-10-12 at 1 18 37 PM</td>
<td>Screenshot 2022-10-12 at 1 56 15 PM</td>
</tr>
</table>

3. Refactors out the tag styling into CSS classes (in so doing, also updates the background color of the tags in the `CommissionDetailsPane`
<table>
<tr>
<td><strong>before</strong></td>
<td><strong>after</strong></td>
</tr>

<tr>
<td>Screenshot 2022-10-12 at 1 08 57 PM</td>
<td>Screenshot 2022-10-12 at 2 15 16 PM</td>
</tr>
</table>

4. Sets a minimum height to the Iterations `ListView` so that you can still see the iterations even if the Commission details are very lengthy
<table>
<tr>
<td><strong>before</strong></td>
<td><strong>after</strong></td>
</tr>

<tr>
<td></td>
<td></td>
</tr>
</table>

5. Update the Commission and Customer Cards
<table>
<th colspan="2">Customer</th>
<tr>
<td><strong>before</strong></td>
<td><strong>after</strong></td>
</tr>

<tr>
<td>Screenshot 2022-10-12 at 4 02 20 PM</td>
<td>Screenshot 2022-10-12 at 4 01 35 PM</td>
</tr>
</table>

<table>
<th colspan="2">Commission</th>
<tr>
<td><strong>before</strong></td>
<td><strong>after</strong></td>
</tr>

<tr>
<td>Screenshot 2022-10-12 at 4 02 24 PM</td>
<td>Screenshot 2022-10-12 at 4 01 49 PM</td>
</tr>
</table>

6. Style Commissions Card (weird behaviour when using image/ circle in Commissions Card, so I used a Label instead)
Screenshot 2022-10-12 at 9 58 23 PM

7. Style the description field conditionally
<table>
<th>no description</th><th>with description</th>
<tr>
<td>Screenshot 2022-10-12 at 11 18 08 PM</td>

<td>Screenshot 2022-10-12 at 11 18 52 PM</td>
</tr>
<table>

### CODE REVIEW

File: src/main/java/seedu/address/ui/CommissionDetailsPane.java [hunk 1]
```diff
@@ -124,6 +132,7 @@ private void updateProgressComponents(Commission commission) {
 
         Commission.CompletionStatusString completionStatusString = commission.getCompletionStatusString();
```
Comment (ngxingyu) [inline src/main/java/seedu/address/ui/CommissionDetailsPane.java:133]: commission might be `null` here

File: src/main/java/seedu/address/ui/CommissionDetailsPane.java [hunk 2]
```diff
@@ -110,29 +111,37 @@ private void setFieldIcons() {
...
+            switch (completionStatusString) {
+            case COMPLETED:
+                completionStatusCircle.setFill(COMPLETED_COLOR);
+                break;
+            case IN_PROGRESS:
+                completionStatusCircle.setFill(IN_PROGRESS_COLOR);
+                break;
+            default:
+                completionStatusCircle.setFill(NOT_STARTED_COLOR);
+            }
```
Comment (seelengxd) [inline src/main/java/seedu/address/ui/CommissionDetailsPane.java:144]: Here as well.

**Review by seelengxd — COMMENTED** [review]
Looks mostly good to me - minor nits for code quality. Thanks for refactoring the tags to use css instead! :P

**Comment by chloeelim** [comment]
Great eye for detail @seelengxd, really appreciate your help!

**Review by seelengxd — APPROVED** [review]
LGTM!

**Pull request merged**